content/news/028/index.md

```diff
@@ -135,6 +135,46 @@ They also worked on a [tutorial][bitgun-tutorial].
 [bitgun-missions]: https://twitter.com/LogLogGames/status/1464009563976392713?s=20
 [bitgun-tutorial]: https://twitter.com/LogLogGames/status/1461898845810348033?s=20
 
+### [Veloren][veloren]
+
+![Travelling merchant](veloren.jpg)
+_A merchant paused in their travels_
+
+[Veloren][veloren] is an open world, open-source voxel RPG inspired by Dwarf
+Fortress and Cube World.
+
+In November, Veloren started a new initiative to help new developers learn more
+about the code base; the Veloren Reading Club. You can watch the
+[first][veloren-reading-club-1] and [second][veloren-reading-club-2] sessions
+now. Hats were merged, and  a major rewrite of the server-hosting section of the
+book happened. New textures were made for item drops. Some concept art was
+created for what massive cities could looks like. Dynamic weather was added, and
+you can [watch that in action][veloren-rain-storm].
+
+Experience sharing went through a large overhaul to evenly distribute EXP gained
+across groups. The difficulty of dungeons was adjusted to be more balanced for
+new players. Persistence was added to skills, and measures were put in place to
+help with future migrations to new skill trees. Site2 works was done to help
```

Comment:
Would it be possible to clarify what `site2` actually is? I thought this was a typo at first, but it sounds like it's part of the world generation systems?

Reply:
Ya, I didn't want to dive too deep into topics, but I can definitely swap this out for a short sentence :)